src/cljs/weasel/impls/websocket.cljs

```diff
@@ -20,7 +20,7 @@
   ([auto-reconnect?]
      (websocket-connection auto-reconnect? nil))
   ([auto-reconnect? next-reconnect-fn]
-     (goog.net.WebSocket. auto-reconnect? next-reconnect-fn)))
```

Comment:
This is great and obviously would work but have you also considered the following?
```clojure
(goog.net.WebSocket. (js-obj "autoReconnect" auto-reconnect?) next-reconnect-fn)
```